### PR TITLE
refactor: update form

### PR DESCRIPTION
**Pull request by juaan**
## Sumary

- [x] update form: remove category field
- [x] update upload image ui

### CODE REVIEW

File: components/UploadForm.tsx [hunk 1]
```diff
@@ -59,38 +42,22 @@ const FormUpload: React.FC<Props> = (props) => {
...
             accept="image/*,.pdf"
+            ref={inputRef}
+            style={{ display: 'none' }}
+          />
+          <FormLabel>{t('components.uploadForm.file.label')}</FormLabel>
+          <Input
+            onClick={() => inputRef?.current?.click()}
+            value={
+              files?.item(0)?.name ||
+              t('components.uploadForm.file.placeholder').toString()
```
Comment (adeyahya) [inline components/UploadForm.tsx:58]: `.toString()` here is not necessary

**Review by adeyahya — COMMENTED** [review]
minor suggestion

File: components/UploadForm.tsx (hunk 1 above)
Comment (juaan) [reply components/UploadForm.tsx:58]: the linter is ngamuk when I didn't put that, do you have any suggestion

Comment (juaan) [reply components/UploadForm.tsx:58]: @adeyahya @adeyahya @adeyahya @adeyahya @adeyahya @adeyahya @adeyahya @adeyahya @adeyahya @adeyahya @adeyahya @adeyahya

**Pull request merged**